### PR TITLE
To-do-list

### PR DESCRIPTION
**Pull request by diegoh40**
in this project, I implemented modular workflow optimization tools such as webpack, created a page dynamically with JS, and styled it with CSS.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,16 @@
```
Comment (danmainah) [inline README.md:5]: - [ ] Kindy change your description to reflect the status of this milestone, it's supposed to be an individual project, and going by your commits you did it alone. Also, do the same to your pull request summary.

- [ ] Kindly check your getting started section remove the unnecessary titles or complete the section it's making your readme look unprofessional with incomplete sections.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @diegoh40, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by diegoh40** [comment]
At the suggestion of the CR I reviewed the Readme file and made the indicated changes, I also adapted the description of my project to a less collective mode and more as individual work.

File: README.md [hunk 2]
```diff
@@ -2,18 +2,20 @@
...
+- Webpack.
+- JS HTML CSS.
 
 ## Live Demo (if available)
 [Live Demo Link](https://livedemo.com)
+
+
 ## Getting Started
 
+
```
Comment (danmainah) [inline README.md]: - [ ] @diegoh40  as indicated earlier kindly remove the unnecessary headings in your getting started section eg prerequisites, setup install. You could also choose to finish the section by adding content to your getting started section.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @diegoh40, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by diegoh40** [comment]
I corrected the readme by removing unnecessary spaces to make the text more understandable.

**Review by AnselemOdims — APPROVED** [review]
### Hi @diegoh40 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**